components/workflow.rst

```diff
@@ -5,8 +5,8 @@
 The Workflow Component
 ======================
 
-    The Workflow component provides tools for managing a workflow or finite state
-    machine.
+    The Workflow component provides tools for managing a workflow or finite
```

Comment:
isn't enough clear to me right now, but it's not a problem in the docs: I just need to learn a little bit more about all of theses implementations ^^

Thank you @stof